backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepositoryTest.java

```diff
@@ -414,6 +414,9 @@ void deleteOrgAndFacilities() {
 
     _repo.deleteOrganization(ABC);
 
+    assertThrows(
```

Comment:
thanks for remembering to add this -- I totally forgot about it 😅 🙌

Reply:
ooo also saw there's a code smell for that haha
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&pullRequest=7828&id=CDCgov_prime-data-input-client&open=AZAoEEefLIlBZf8MjHLN on `line 419`